Docs/getting-started/try-it-out.md

```diff
@@ -2,6 +2,12 @@
 
 If you want to give Singularity a try, you can install [docker](https://docs.docker.com/installation/) and [docker-compose](https://docs.docker.com/compose/#installation-and-set-up) to run our example cluster.
 
+**Note** If you are running Docker for macOS you need to run Singularity on a docker machine. You can create one with the following commands.
```

Comment:
Just to be specific:

**Note** The Docker development setup is currently not compatible with Docker for Mac due to its usage of `HOST` network mode. You can instead run Singularity via `docker-machine` using the following commands: